resources/ci_cd/linux_format.py

```diff
@@ -97,7 +97,7 @@ def get_argument_parser() -> ArgumentParser:
         help="Disable markdownlint linter.",
     )
     parser.add_argument(
-        "-cxx",
+        "-clang",
```

Comment:
we need to update the contributing.md accordingly

Reply:
Done 👍🏻